### PR TITLE
Mappings first stab

### PR DESCRIPTION
**Pull request by hrshdhgd**
- [x] Add `Makefile`
- [x] Add mappings for: MONDO
    - [x] DO
    - [x] NCIT
    - [x] ORDO
    - [ ] OMIM+OMIMPS

### CODE REVIEW

**Comment by matentzn** [comment]
Good direction :)

**Comment by hrshdhgd** [comment]
@matentzn I have added NCIT-ICD10 mappings as seen [here](https://github.com/mapping-commons/disease-mappings/blob/new_mappings/mappings/ncit-icd10.sssom.tsv). I have added placeholders for `IMDRF` as you can see in the `curie_map` for now. I wasn't sure if there is another way to get that info.
Thoughts?

**Comment by matentzn** [comment]
Awesome man, so happy seeing this shap up! I am suprised at how small the NCIT-ICD mapping is.. the raw data has the same number of rows?

**Comment by hrshdhgd** [comment]
Yup ~800 from `ICD10CM 2017` and ~15 from `ICD10CM 2016`

**Comment by matentzn** [comment]
Lol ok, that aint very impressive :D Thanks!

**Comment by matentzn** [comment]
(I assume we are reasonably certain the website didn't have other mappings served under other links?)

**Comment by hrshdhgd** [comment]
If they do, the title of the file does not state the same. I can double check.

**Comment by hrshdhgd** [comment]
`ICD9CM` and `ICDO3`. Are these relevant?

**Comment by matentzn** [comment]
No! But thanks for asking.

**Comment by matentzn** [comment]
Would you be able to create a google sheet to capture all the prefixes you come across? Just informal, maybe something like

| Prefix | Description | Example |
| ----- | --------- | ------- |
| ICD9CM | ICD9 Clinical modification | ICD9CM:X |

Something like that. No sweat but I would  like a handle on all the different prefixes we handle in disease mapping commonns.

**Comment by hrshdhgd** [comment]
I have It in [metadata.yaml](https://github.com/mapping-commons/disease-mappings/blob/dc15488e30b9c25ebbde6b090a39fd29a32b66c0/metadata.yaml#L1388) below `# Added by me`. But I can make a Google Sheet tracking these. Which common domain do you want me to save the sheet?

**Comment by matentzn** [comment]
Slightly orthogonal! No worries, using the context you made is great.

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,56 @@
+MAPPINGS_DIR=mappings
+OMIM=https://github.com/monarch-initiative/omim/releases/download/latest/omim.ttl
+ORDO=http://www.orphadata.org/data/ORDO/ORDO_en_4.0.owl
```
Comment (cmungall) [inline Makefile:3]: we have other parts of the mondo pipeline that does some ETL of the ORDO OWL, should we use that?

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,56 @@
+MAPPINGS_DIR=mappings
+OMIM=https://github.com/monarch-initiative/omim/releases/download/latest/omim.ttl
```
Comment (cmungall) [inline Makefile:2]: is this what @joeflack4 is using for slurps?

**Review by cmungall — COMMENTED** [review]
don't check in pyc files

I assume we will coordinate later with the mondo slurp pipeline

File: Makefile (hunk 1 above)
Comment (matentzn) [reply Makefile:3]: For this particular problem here, I think I prefer to have a 1:1 reflection of the mappings as they are in the source ontologies, with no prefix changing, interpretations and so on. If we go through an intermediate, then we open ourselves to questions of obscure provenance (The ORDO owl we use for ingestion is massaged in certain ways so we can ingest it). Right now my priority is on starting the boomer process, and then iterate - if in one of these iterations we decide we _have_ to massage ORDO before sssom parse, then we come back to your question.. Sound ok?

File: mappings/ncit_icd10_2017.sssom.tsv [hunk 3]
```diff
@@ -0,0 +1,809 @@
+# creator_id: h2
+# curie_map:
+#   ICD10CM: http://apps.who.int/classifications/icd10/browse/2010/en#/
+#   NCI: http://purl.obolibrary.org/obo/NCIT_
+# license: https://creativecommons.org/publicdomain/zero/1.0/
+# mapping_date: '2018-07-05'
+# mapping_set_id: http://w3id.org/mapping_commons/disease-mappings/onto-icd10
```
Comment (matentzn) [inline mappings/ncit_icd10_2017.sssom.tsv:7]: change to http://w3id.org/mappings/disease-mappings/ncit_icd10_2017.sssom.tsv

File: mappings/ncit_icd10_2017.sssom.tsv [hunk 4]
```diff
@@ -0,0 +1,809 @@
+# creator_id: h2
```
Comment (matentzn) [inline mappings/ncit_icd10_2017.sssom.tsv:1]: Remove creator_id for now

File: mappings/ncit_icd10_2017.sssom.tsv [hunk 5]
```diff
@@ -0,0 +1,809 @@
...
+# curie_map:
+#   ICD10CM: http://apps.who.int/classifications/icd10/browse/2010/en#/
+#   NCI: http://purl.obolibrary.org/obo/NCIT_
+# license: https://creativecommons.org/publicdomain/zero/1.0/
+# mapping_date: '2018-07-05'
+# mapping_set_id: http://w3id.org/mapping_commons/disease-mappings/onto-icd10
+# mapping_tool: https://github.com/mapping-commons/sssom-py
```
Comment (matentzn) [inline mappings/ncit_icd10_2017.sssom.tsv:8]: Add `mapping_provider: URLWHEREITWASDOWNLOADEDFROM`

File: mappings/ncit_icd10_2016.sssom.tsv [hunk 6]
```diff
@@ -0,0 +1,20 @@
+# creator_id: h2
```
Comment (matentzn) [inline mappings/ncit_icd10_2016.sssom.tsv:1]: same comments as 2017 mapping set

**Review by matentzn — APPROVED** [review]

File: mappings/ncit_icd10_2017.sssom.tsv (hunk 4 above)
Comment (hrshdhgd) [reply mappings/ncit_icd10_2017.sssom.tsv]: Done

File: mappings/ncit_icd10_2016.sssom.tsv (hunk 6 above)
Comment (hrshdhgd) [reply mappings/ncit_icd10_2016.sssom.tsv]: Done

File: mappings/ncit_icd10_2017.sssom.tsv (hunk 3 above)
Comment (hrshdhgd) [reply mappings/ncit_icd10_2017.sssom.tsv]: Done

**Review by matentzn — APPROVED** [review]

**Pull request merged**